tests/command_test.py

```diff
@@ -14,11 +14,23 @@
 
 
 class TestTest(unittest.TestCase):
+    def extract_json_data_from_bytes_array(self, input_bytes):
```

Comment:
```suggestion
    def extract_json_data_from_bytes_array(self, input_bytes: bytes) -> bytes:
```

nitpick: 型ヒントを書いておいてほしい

(周囲のコードは古いので書いてないのですが (たぶんまわりに合わせたのだと思います。すまない :bow:)、新規に書くコードにはできるだけ書いていきたい)